File: cpp/bc1132.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal, sum = 0, aux;
	
	cin >> fVal >> sVal;
	
	if(fVal > sVal)
	{
		aux = fVal;
		fVal = sVal;
		sVal = aux;
	}
	
	for(int i = fVal; i < sVal; i++)
		if(i % 13 != 0) sum += i;
	
	cout << sum << endl;
}

File: python/bc1132.py
fVal = int(input())
sVal = int(input())
su = 0

if fVal > sVal:
    for c in range(sVal, fVal):
        if c % 13 != 0: su += c
else:
    for c in range(fVal, sVal):
        if c % 13 != 0: su += c

print(su)